src/main/java/de/zalando/aruha/nakadi/managers/Result.java

```diff
@@ -0,0 +1,82 @@
+package de.zalando.aruha.nakadi.managers;
+
+import org.zalando.problem.Problem;
+
+import javax.ws.rs.core.Response;
+
+public interface  Result<T> {
```

Comment:
Could you please describe in more details? It could either have result either problem which occurred.